CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Added
 
 ### Changed
+- Add `AbstractHtmlProcessor::fromHtml`
+  ([#675](https://github.com/MyIntervals/emogrifier/pull/675))
```

Comment:
Shouldn't this be in the "Added" section?

Reply:
You're perfectly right. Fixed and repushed.